engine/robot_state.py

```diff
@@ -115,60 +118,79 @@ def __init__(self, **kwargs):
+        # dividing by time_step ignores the effect of time_step on absolute
+        self.turn_angle = self.turn_angle / self.time_step
         self.init_threshold = kwargs.get("init_threshold", 1)
         self.goal_threshold = kwargs.get("goal_threshold", 1)
         self.noise_margin = kwargs.get("noise_margin", 1)
-        self.front_sensor_offset = kwargs.get("front_sensor_offset", 0)  # TODO: replace this with how far offset the sensor is to the front of the robot
+        # TODO: replace this with how far offset the sensor is to the front of the robot
+        self.front_sensor_offset = kwargs.get("front_sensor_offset", 0)
         self.max_sensor_range = kwargs.get("max_sensor_range", 600)
         self.sensor_measuring_angle = kwargs.get("sensor_measuring_angle", 75)
-        self.width_margin = kwargs.get("width_margin", 1)  # TODO: replace this with actual margin
-        self.threshold_distance = kwargs.get("threshold_distance", ((self.width + self.width_margin) / 2) / math.cos(math.radians((180 - self.sensor_measuring_angle) / 2)))
-        self.detect_obstacle_range = kwargs.get("detect_obstacle_range", min(self.threshold_distance, self.max_sensor_range))  # set ultrasonic detection range
+        # TODO: replace this with actual margin
+        self.width_margin = kwargs.get("width_margin", 1)
+        self.threshold_distance = kwargs.get("threshold_distance", ((
+            self.width + self.width_margin) / 2) / math.cos(math.radians((180 - self.sensor_measuring_angle) / 2)))
+        self.detect_obstacle_range = kwargs.get("detect_obstacle_range", min(
+            self.threshold_distance, self.max_sensor_range))  # set ultrasonic detection range
 
         # MEASUREMENTS
-        self.state = kwargs.get("state", np.array([[kwargs.get("xpos", 0)], [kwargs.get("ypos", 0)], [kwargs.get("heading", 0)]]))
-        self.truthpose = kwargs.get("truthpose", np.transpose(np.array([[kwargs.get("xpos", 0)], [kwargs.get("ypos", 0)], [kwargs.get("heading", 0)]])))
+        self.state = kwargs.get("state", np.array(
+            [[kwargs.get("xpos", 0)], [kwargs.get("ypos", 0)], [kwargs.get("heading", 0)]]))
+        self.truthpose = kwargs.get("truthpose", np.transpose(np.array(
+            [[kwargs.get("xpos", 0)], [kwargs.get("ypos", 0)], [kwargs.get("heading", 0)]])))
         self.plastic_level = kwargs.get("plastic_level", 0)
         self.battery = kwargs.get("battery", 100)
         self.acceleration = kwargs.get("acceleration", [0, 0, 0])
         self.magnetic_field = kwargs.get("magnetic_field", [0, 0, 0])
         self.gyro_rotation = kwargs.get("gyro_rotation", [0, 0, 0])
         self.init_gps = kwargs.get("init_gps", (0, 0))
-        self.gps_data = kwargs.get("gps_data",(0, 0) )
-        self.imu_data = kwargs.get("imu_data", None)  # will be filled by execute_setup
+        self.gps_data = kwargs.get("gps_data", (0, 0))
```

Comment:
can we put the comments above the variable instead of under? (or are they next to? i cant really tell from the github ui)